.github/workflows/run_system_tests.yml

```diff
@@ -28,22 +24,44 @@ jobs:
         uses: actions/checkout@v4
       - name: Copy and rename .env.simulation to .env
         run: cp examples/.env.simulation .env
-        working-directory: ./
-      - name: Cache virtualenvs
+
+      # ni-measurement-plugin-sdk-service  
+      - name: Cache virtualenv (ni-measurement-plugin-sdk-service)
         uses: actions/cache@v4
-        id: cache
+        id: cache-nims
         with:
           path: |
             packages/service/.venv
             packages/service/.tox
-          key: run-system-tests-${{ runner.os }}-${{ matrix.configuration }}-${{ hashFiles('packages/service/poetry.lock') }}
-      - name: Install dependencies
+          key: run-system-tests-nims-${{ runner.os }}-${{ matrix.configuration }}-${{ hashFiles('packages/service/poetry.lock') }}
+      - name: Install dependencies (ni-measurement-plugin-sdk-service)
+        run: poetry install -v
+        working-directory: ./packages/service
+      - name: Run system tests (ni-measurement-plugin-sdk-service)
+        run: poetry run tox
+        working-directory: ./packages/service
+
+      # ni-measurement-plugin-sdk-generator
+      - name: Cache virtualenv (ni-measurement-plugin-sdk-generator)
+        uses: actions/cache@v4
+        id: cache-nimg
+        with:
+          path: |
+            packages/generator/.venv
+            packages/generator/.tox
+          key: run-system-tests-nimg-${{ runner.os }}-${{ matrix.configuration }}-${{ hashFiles('packages/generator/poetry.lock') }}
+      - name: Install dependencies (ni-measurement-plugin-sdk-generator)
         run: poetry install -v
-      - name: Run system tests
+        working-directory: ./packages/generator
+      - name: Run system tests (ni-measurement-plugin-sdk-generator)
         run: poetry run tox
+        working-directory: ./packages/generator
+
       - name: Upload test results
         uses: actions/upload-artifact@v4
         with:
           name: test_results_system_${{ matrix.configuration }}
-          path: ./packages/service/test_results/*.xml
+          path: |
+            ./packages/generator/test_results/*.xml
+            ./packages/service/test_results/*.xml
```

Comment:
These aren't getting reported. Compare this PR to https://github.com/ni/measurement-plugin-python/actions/runs/10910937380

https://github.com/ni/measurement-plugin-python/actions/runs/10910937380 :
> Test Results
> 
>    30 files      30 suites   38m 31s ⏱️
>   665 tests    665 ✅     0 💤 0 ❌
>16 550 runs  15 480 ✅ 1 070 💤 0 ❌
>
>Results for commit e1127d71.

This PR: 
> Test Results
>    20 files  ±0     20 suites  ±0   16m 55s ⏱️ +11s
>   516 tests ±0    516 ✅ ±0      0 💤 ±0  0 ❌ ±0 
>10 040 runs  ±0  8 980 ✅ ±0  1 060 💤 ±0  0 ❌ ±0 
>
>Results for commit 8228c47e. ± Comparison against earlier commit 63516d5e.

The number of files, tests, and runs went down significantly, but we are adding more configurations for py32 vs py64, so these should have gone up.

I think the problem is that adding more directories to the artifact changes its internal directory structure:

https://github.com/actions/upload-artifact
> If multiple paths are provided as input, the least common ancestor of all the search paths will be used as the root directory of the artifact. Exclude paths do not affect the directory structure.

https://github.com/ni/measurement-plugin-python/actions/runs/10910937380 :
![image](https://github.com/user-attachments/assets/92c09859-3eaa-44f2-8e58-27066d34e598)

This PR: 
![image](https://github.com/user-attachments/assets/fb0aed97-42d1-46a5-b0ca-00afe327dd89)
![image](https://github.com/user-attachments/assets/93203bca-413b-48ce-8ca3-9131f7166cf5)

Please change https://github.com/ni/measurement-plugin-python/blob/main/.github/workflows/report_test_results.yml to use `test_results/**/*.xml` so it recurses into subdirectories.

Reply:
Update:
> Test Results
>    40 files  +   20      40 suites  +20   50m 6s ⏱️ + 33m 11s
>   686 tests +  170     686 ✅ +  170      0 💤 ± 0  0 ❌ ±0 
>16 760 runs  +6 720  15 690 ✅ +6 710  1 070 💤 +10  0 ❌ ±0 
>
>Results for commit 71ad5fb8. ± Comparison against earlier commit 8228c47e.